### PR TITLE
Add a turbodbc recipe

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

File: recipes/turbodbc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  summary: 'access relational databases via the Open Database Connectivity (ODBC) interface'
+  description: |
+    Turbodbc is a Python module to access relational databases via the Open
+    Database Connectivity (ODBC) interface. In addition to complying with the
+    Python Database API Specification 2.0, turbodbc offers built-in NumPy
+    support. Don't wait minutes for your results, just blink.
+
+extra:
+  recipe-maintainers:
+    - MathMagique
```
Comment (xhochy) [inline recipes/turbodbc/meta.yaml]: @MathMagique Can you ack that this is ok for you?

**Comment by xhochy** [comment]
Build failure on Linux is an upstream problem, will retrigger when fixed upstream.

**Comment by xhochy** [comment]
After two upstream releases, everything is building fine now :) Ready for review.

File: recipes/turbodbc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  run:
+    - boost 1.63.*
+    - numpy x.x
+    - python
+    - six
+    - unixodbc
+
+test:
+  imports:
+    - turbodbc
```
Comment (ericdill) [inline recipes/turbodbc/meta.yaml]: Please add this to the test section:
```yaml
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```

Comment (xhochy) [inline recipes/turbodbc/meta.yaml]: Done.

**Pull request merged**